<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1401</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Imagenes.qrc">:/Imagenes/fondo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 40pt &quot;Times New Roman&quot;;
background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>7</number>
    </property>
    <property name="text">
     <string>MrDelivery</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
font: italic 14pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
font: italic 14pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>10</number>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_avanzar">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>550</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 85, 0);
font: italic 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>SELECCIONAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_VerGrafo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 85, 0);
font: italic 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>VER GRAFO</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>   DESENCOLAR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>       ACTUAL</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_CostosDesde1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 85, 0);
font: italic 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>COSTOS DESDE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CostosDesde2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 85, 0);
font: italic 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>TODOS LOS CAMINOS</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>COSTO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MINUTOS</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_HayCamino">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 85, 0);
font: italic 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>¿HAY CAMINO?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CaminoOptimo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 85, 0);
font: italic 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>CAMINO ÓPTIMO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_profundidad">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 85, 0);
font: italic 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>PROFUNDIDAD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_anchura">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 85, 0);
font: italic 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>ANCHURA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AExpaMini">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 85, 0);
font: italic 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>A. EXPANSIÓN MÍNIMA</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Cola">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Times New Roman&quot;;
background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>7</number>
    </property>
    <property name="text">
     <string>EN COLA: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Cola_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Times New Roman&quot;;
background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>7</number>
    </property>
    <property name="text">
     <string>PONDERACIÓN:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>50</y>
      <width>401</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
    </property>
    <property name="title">
     <string>GRAPH MANAGER</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>361</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);
font: italic 14pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>10</number>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Activar">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>420</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 85, 0);
font: italic 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>ACTIVAR/DESACTIVAR VÉRTICE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Activar_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>370</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 85, 0);
font: italic 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>ACTIVAR/DESACTIVAR ARISTA</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 16pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Activar_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(255, 85, 0);
font: italic 14pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>MOSTRAR GRAFO MODIFICADO</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 16pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox_8">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>550</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Times New Roman&quot;;
color: rgb(255, 85, 0);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
